inc/luos_list.h

```diff
@@ -90,6 +91,12 @@ typedef enum
     PARAMETERS, // depend on the service, can be : servo_parameters_t, imu_report_t, motor_mode_t
     ERROR_CMD,
 
+    // Monitoring commands
+    GET_RTB,
```

Comment:
This is again for the communication with the exterior

Reply:
ok I get it, this is because you need to be able to send a RTB from the user side and for now only Luos can make it.
Is it not possible to move the Luos command to the user command side?